src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -77,7 +77,6 @@ const OptionRowLHN = (props) => {
     const hoveredBackgroundColor = props.hoverStyle && props.hoverStyle.backgroundColor ? props.hoverStyle.backgroundColor : themeColors.sidebar;
     const focusedBackgroundColor = styles.sidebarLinkActive.backgroundColor;
 
-    const avatarTooltips = !optionItem.isChatRoom && !optionItem.isArchivedRoom ? _.pluck(optionItem.displayNamesWithTooltips, 'tooltip') : undefined;
```

Comment:
Now we have `shouldShowTooltip` prop. Please add that logic
`shouldShowTooltip={!optionItem.isChatRoom && !optionItem.isArchivedRoom}`